dark/graphics.py

```diff
@@ -667,7 +667,10 @@ def alignmentPanelHTML(titlesAlignments, sortOn='maxScore',
         if not S_ISDIR(os.stat(outputDir).st_mode):
             raise ValueError("%r is not a directory." % outputDir)
     else:
-        os.mkdir(outputDir)
+        if outputDir == None:
+            raise NameError("The outputDir needs to be specified.")
```

Comment:
And you should raise a `ValueError` and document it in the docstring at the start of the function.

Reply:
done